jwst/pipeline/calwebb_detector1.cfg

```diff
@@ -4,34 +4,19 @@ save_calibrated_ramp = False
 
     [steps]
       [[group_scale]]
-        config_file = group_scale.cfg
       [[dq_init]]
-        config_file = dq_init.cfg
       [[saturation]]
-        config_file = saturation.cfg
       [[ipc]]
         skip = True
```

Comment:
Is there a reason we even have `IPCStep` in the pipeline if it is skipped?

Reply:
We left it in the code so that it's at least available should anyone want to run it on their own. There are ref files for some instruments. But for now it's skipped by default in normal Ops processing.